pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -108,11 +114,12 @@ func validateFilters(filters []string, fieldName string) (causes []metav1.Status
 		return nil
 	}
 
+	errPattern := "%s filter %s is invalid: cannot contain a %s character (%s); FilterRules: %s"
```

Comment:
nit: Keep definition of this at L129 and as `const`.

Reply:
get it!

Reply:
done